org.eclipse.lsp4e.test/src/org/eclipse/lsp4e/test/references/FindReferencesTest.java

```diff
@@ -74,11 +78,14 @@ public void log(UiFreezeEvent event) {
 
 	@Before
 	public void setUp() throws CoreException {
+		shell = new Shell();
```

Comment:
Can't we use `textViewer.getTextWidget().getDisplay()` directly here instead of instantiating a new shell?

Reply:
I think so